src/Logary/SpanModule.fs

```diff
@@ -90,7 +90,7 @@ module Span =
             if not hasFired then
               do hasFired <- true
               do activeSpan.Value <- previous
-              logger.logWith Info composedMsgFac
+              logger.logWithBP Info composedMsgFac |> Hopac.Hopac.start
```

Comment:
since we change `logWith` using `queue` not `start`, the `messageFac` will be invoked on hopac's thread not user's callsite thread. 

generally, span here will heavily depend on `ambientSpanId` middleware, it needs to run on user's callsite thread, so change here  to use `logWithBP + start`

Reply:
and the `logWith` api return a `unit`, like some fire-forget action, so using `queue` will gain some more perf if hopac's threads are busy.